src/ctia/auth/threatgrid.clj

```diff
@@ -70,8 +70,8 @@
                                                    (get default-capabilities))}))))
         auth/denied-identity-singleton)))
 
-(defn make-auth-service []
-  (let [{:keys [whoami-url cache]} (p/get-in-global-properties [:ctia :auth :threatgrid])
+(defn make-auth-service [get-in-config lookup-stored-identity]
```

Comment:
I would rename `lookup-stored-identity` into `lookup-identity`. This function actually ignore the store details.

Reply:
I added this to the epic and I'll do this after everything is merged.